string object_type
string sub_type
---
bool result
